src/web/detectors/RotationGestureDetector.ts

```diff
@@ -18,10 +18,10 @@ export default class RotationGestureDetector
   private previousTime = 0;
 
   private previousAngle = 0;
-  private rotation = 0;
+  private _rotation = 0;
```

Comment:
Why were some private fields like `rotation` converted into a `getter`+`setter`, while other like `previousAngle` weren't?

Reply:
Because rotation [is used outside of RotationGestureDetector](https://github.com/software-mansion/react-native-gesture-handler/blob/56c33510127b26386e5c46cfce8019bcd0b7e61e/src/web/handlers/RotationGestureHandler.ts#L21)

Reply:
You're right, I missed that.